layouts/black-lives-matter.hbs

```diff
@@ -0,0 +1,138 @@
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta name="description" content="#BlackLivesMatter">
+
+  <link rel="apple-touch-icon" href="https://nodejs.org/static/images/favicons/apple-touch-icon.png" sizes="180x180">
+  <link rel="icon" href="https://nodejs.org/static/images/favicons/favicon-32x32.png" sizes="32x32" type="image/png">
+  <link rel="icon" href="https://nodejs.org/static/images/favicons/favicon-16x16.png" sizes="16x16" type="image/png">
+  <link rel="manifest" href="https://nodejs.org/static/images/favicons/manifest.json">
+  <link rel="mask-icon" href="https://nodejs.org/static/images/favicons/safari-pinned-tab.svg" color="#43853d">
+  <link rel="shortcut icon" href="https://nodejs.org/static/images/favicons/favicon.ico">
+  <meta name="msapplication-TileColor" content="#333333">
+  <meta name="msapplication-config" content="https://nodejs.org/static/images/favicons/browserconfig.xml">
+  <meta name="theme-color" content="#333333">
+
+  <meta property="og:site_name" content="Node.js">
+  <meta property="og:title" content="#BlackLivesMatter">
+  <meta property="og:image" content="https://nodejs.org/static/images/logo-hexagon-card.png">
+  <meta property="og:image:type" content="image/png">
+  <meta property="og:image:width" content="224">
+  <meta property="og:image:height" content="256">
+  <meta property="og:description" content="#BlackLivesMatter">
+
+  <meta name="twitter:card" content="summary">
+  <meta name="twitter:site" content="@nodejs">
+  <meta name="twitter:title" content="#BlackLivesMatter">
+  <meta name="twitter:image" content="https://nodejs.org/static/images/logo-hexagon-card.png">
+  <meta name="twitter:image:alt" content="The Node.js Hexagon Logo">
+  <style>
+    body {
+      background: #333;
+      color: #fff;
+      min-width: 80%;
+      max-width: 50%;
+      margin: 0 auto;
+    }
+
+    section {
+      padding-bottom: 30px;
+    }
+
+    ul.humans > li {
+      list-style: none;
+      font-size: 1.7rem
+    }
+
+    h1 {
+      font-size: 5rem
+    }
+
+    .center {
+      margin: 0 auto
+    }
+
+    .space {
+      padding-top: 4rem;
+      padding-bottom: 2rem;
+    }
+
+    footer {
+      margin-bottom: 2rem;
+    }
+  </style>
+</head>
+
+<body>
+  <h1 class="center space">#BlackLivesMatter</h1>
+  <section id="lives" class='center'>
+    <ul class="humans">
+      <li>George Floyd</li>
+      <li>Ahmaud Arbery</li>
+      <li>Mike Ramos</li>
+      <li>Tamir Rice</li>
+      <li>Michael Brown</li>
+      <li>Freddie Gray</li>
+      <li>Eric Garner</li>
+      <li>Philando Castile</li>
+      <li>Anton Sterling</li>
+      <li>Delrawn Small</li>
+      <li>Sandra Bland</li>
+      <li>Botham Jean</li>
+      <li>Trayvon Martin</li>
+      <li>Jamar Clark</li>
+      <li>Laquan McDonald</li>
+      <li>Antwon Rose II</li>
+      <li>Keith Childress</li>
+      <li>Walter Scott</li>
+      <li>Alton Sterling</li>
+      <li>Stephon Clark</li>
```

Comment:
below this last list item i think its also important to acknowledge that many countless names did not get attention and are no less notable or valuable - something like 
> <li>and countless more names not listed here whose lives were taken by violence and brutality</li>

what do you think?

Reply:
+1

Reply:
James opened a PR on the repo I was doing this work in along those lines, too, fwiw.

Reply:
Alton Sterling and Stephon Clark are repeated earlier in the list

Reply:
fixed